docs/source/azure/configuration/index.rst

```diff
@@ -12,6 +12,17 @@ Azure Functions and monitoring the results of policies.
   :glob:
 
   authentication
+  monitoring
```

Comment:
I know you didn't change the intro to this page, but the intro sentence on this page needs to be changed to be inclusive of functions and container hosting. Maybe have a quick discussion of when I would want to use which one